src/App.tsx

```diff
@@ -30,12 +35,15 @@ const App = () => {
         });
     const history = useHistory();
     function goHome() {
-        history.push("/claim/");
+        history.push("/account/");
     }
     const { accountInformation } = useApplicationState();
     useEffect(() => {
-        if (accountInformation.state === AccountState.notSet) history.replace('/claim/');
-        if ([AccountState.valid].includes(accountInformation.state!)) history.replace('/claim/'+accountInformation.account!.id);
+        if (history.location.pathname.startsWith('/claim/') || history.location.pathname.startsWith('/account/')) {
+            let page = history.location.pathname.split('/')[1];
+            if (accountInformation.state === AccountState.notSet) history.replace(`/${page}/`);
+            if ([AccountState.valid].includes(accountInformation.state!)) history.replace(`/${page}/${accountInformation.account!.id}`);
+        }
```

Comment:
This makes sure only valid accounts work as URL slugs. Previously it was hardcoded to the `/claim/` EP and now additionally works for `/account/` EP.